File: laud-ai/model/domain/Laboratorio.py
from flask_sqlalchemy import SQLAlchemy

db = SQLAlchemy()

class Laboratorio(db.Model):
    id = db.Column(db.Integer, primary_key=True)
    nome = db.Column(db.String(100))
    endereco = db.Column(db.String(200))
    telefone = db.Column(db.String(20))
    exames = db.Column(db.String(200))

File: laudai/src/main/java/br/com/laudai/domain/model/Laboratorio.java
package br.com.laudai.domain.model;

import jakarta.persistence.*;
import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

import java.util.List;

@AllArgsConstructor
@Entity
@NoArgsConstructor
@Getter
@Setter
public class Laboratorio {

    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Integer id;

    private String nome;

    @OneToMany(cascade = CascadeType.ALL, orphanRemoval = true)
    private List<Consulta> consultas;

    @OneToMany(cascade = CascadeType.ALL, orphanRemoval = true)
    private List<Exame> exames;

    @OneToMany(cascade = CascadeType.ALL, orphanRemoval = true)
    private List<Radiologista> radiologista;

}
